package.json

```diff
@@ -23,13 +24,18 @@
 	},
 	"homepage": "https://github.com/atlp-rwanda/phantom-be-codebandits#readme",
 	"devDependencies": {
+		"chai": "^4.3.6",
+		"coveralls": "^3.1.1",
 		"eslint": "^8.12.0",
 		"eslint-config-airbnb-base": "^15.0.0",
+		"eslint-config-prettier": "^8.5.0",
 		"eslint-plugin-import": "^2.25.4",
 		"husky": "^7.0.4",
 		"lint-staged": "^12.3.7",
+		"mocha": "^9.2.2",
+		"nyc": "^15.1.0",
 		"prettier": "2.6.1",
-		"eslint-config-prettier": "^8.5.0"
+		"sinon": "^13.0.1"
```

Comment:
It is better to remove unused devDependencies. for this case, like * coveralls, * nyc, * sinon is doing nothing.

Reply:
I thought Sinon and coveralls dependencies are playing a big role for this PR 
why are they marked unused by deepcheck

Reply:
We are actually using `nyc` and `coveralls` for coverage reporting. `Sinon` is not being used right now but we will need it in testing.